aggregate_test.go

```diff
@@ -335,3 +337,75 @@ func BenchmarkAggregation(b *testing.B) {
 		})
 	}
 }
+
+func Test_Aggregation_DynCol(t *testing.T) {
```

Comment:
Can we write this as a logic test? I think we can reuse the default schema and find the max label or something

Reply:
Would have to add support for aggregation on strings which is why I wrote it as this. Was just more work to write as a logictest